test/test-apimatic-core/utilities/constants.rb

```diff
@@ -0,0 +1,11 @@
+module CoreLibrary
+  CONTENT_TYPE_HEADER = 'content-type'
```

Comment:
Nits, Blocker: These constants should be part of the core library itself and should be accessed from the test package. Same constants should be used in the source package wherever needed.